src/tasks/misc.ts

```diff
@@ -1133,6 +1136,32 @@ export const MiscQuest: Quest = {
       freeaction: true,
       limit: { tries: 2 },
     },
+    {
+      name: "Clan Photo Booth Free Kill",
+      after: [],
+      priority: () => Priorities.Free,
+      ready: () =>
+          Clan.getWhitelisted().find(
+          (c) => c.name === getClanName(),
+          ) !== undefined
+          && get("_photoBoothEquipment", 0) === 0 ,
+      completed: () => (have($item`Sheriff moustache`) && have($item`Sheriff badge`) && have($item`Sheriff pistol`)) || get("_photoBoothEquipment", 0) >= 3,
+      do: (): void => {
+        const currentClan = getClanName();
+        if(
+        (Clan.getWhitelisted().find(
+        (c) => c.name === "Bonus Adventures from Hell",
+        ) === undefined))
+          chatPrivate("bafhbot", "whitelist bafh")
```

Comment:
Don't we need to wait and see if the bot responds? I would almost say this is out of scope, and we can just assume the user has a BAFH whitelist, or skip the photobooth.

Reply:
I'd say definitely don't make them apply for the clan, at most just log a message explaining why they can't join the clan.

Imagine running garbo and finding out it automatically puts all your bags of garbage in store at 210 meat.
Also, not everyone has a whitelist to their own clan so you could be effectively kicking them out of their clan if they're not whitelisted.

Reply:
I check that in the ready condition; you need to be whitelisted to both clans?